File: functions/notes.py
# Chiara Negron Wilson, Function notes

# functions are places that hold actions to be reused
#number= int(input("Please tell me a number:\n"))
def add(numOne,numTwo):# Parameters set the name of the variables( Just for trhe function)
    return numOne+numTwo

    #print (numOne+numTwo)


#print(add(number+21))# arguments set the value of variable.
#print(add(2,100))
#print(add(3,50))

def values(type):
    return input(f"Please give me a {type}:\n")

name =values("name")

place =values("place")

verb =values("verb(past tense)")

print(f"{name} was really fast getting to {place} because they {verb} the whole way there.")

File: functions/notes.c
// Chiara Wilson, Functions Notes in C
#include <stdio.h>
int num;
char name[50],place[50],verb[50];
int add(int numOne,int numTwo){
return numOne+numTwo;



}

void due(char assignment[50], char day[20]){
printf("The %s assignment is due %s\n",assignment,day);




}


int main(void){
 //printf("Please tell me a number\n");
 //scanf("%d",num);
 //add(num,10);
 //add(8,1);
 //add(72,5);
 //printf("%d",add(72,5));
due(" Functions notes", "Today");
due(" Hello world Update", "Tommorow");
due(" Finnacial calc update", "Tommorow");
    return 0;
} 